assets/js/src/core/app/i18n/helper/translation-helper.ts

```diff
@@ -0,0 +1,28 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { store } from '@Pimcore/app/store'
+import { api as translationApi } from '@Pimcore/modules/app/translations/translations-api-slice.gen'
+
+export const addNewTranslations = async (translations: string[]): Promise<void> => {
+  await store.dispatch(translationApi.endpoints.translationUpdate.initiate({
```

Comment:
Just for my interest, how does the endpoint behave, when there is already an available translation on the server.
Will it still override the value?

Reply:
This is a good question - I will double-check that with the BE team

Reply:
As discussed, we should have something like a create endpoint for the missing translations.
Just to ensure, that we not override a translation, that already exists. 
The endpoint should also fail silently in case of an existing key. This key should then be skipped.